app/serializers/node_serializer_data.rb

```diff
@@ -471,12 +645,17 @@ module NodeSerializerData
       }
     },
     cost: {
-      'total_initial_investment_per(:plant)' => { label: 'Initial purchase price', unit: 'EUR' },
-      'cost_of_installing_per(:plant)' => { label: 'Cost of installing', unit: 'EUR' },
-      'decommissioning_costs_per(:plant)' => { label: 'Decommissioning costs', unit: 'EUR' },
-      'fixed_operation_and_maintenance_costs_per(:plant)' => {
+      'total_initial_investment_per(:mw_typical_input_capacity) + cost_of_installing_per(:mw_typical_input_capacity) + decommissioning_costs_per(:mw_typical_input_capacity)' => {
```

Comment:
`Total_initial_investment` already includes cost of installing. I think we can use `total_investment_over_lifetime` to capture initial investment, installing and decomissioning.

Reply:
Fixed this, good idea!